napari/components/dims.py

```diff
@@ -219,6 +223,16 @@ def _check_dims(cls, values):
         elif labels_ndim > ndim:
             updated['axis_labels'] = axis_labels[-ndim:]
 
+        # Check the rollable axes tuple has same number of elements as ndim
+        rollable = values['rollable']
+        n_rollable = len(rollable)
+        if n_rollable < ndim:
+            updated['rollable'] = list(rollable) + (
+                [True] * (ndim - n_rollable)
+            )
+        elif n_rollable > ndim:
+            updated['rollable'] = rollable[:ndim]
```

Comment:
```suggestion
        updated['rollable'] = (values['rollable'] + ([True] * ndim))[:ndim]
```
Single line, but maybe less readable for someone.